extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -667,30 +667,6 @@ static public function onArticleDelete( $article, &$user, &$reason, &$error ) {
 	}
 
 	/**
-	 * @param RecentChange $recentChange
-	 * @return bool
-	 */
-	static public function onRecentChangeSave( $recentChange ) {
```

Comment:
@TK-999 what is the reason for removing this hook? At the first sight it doesn't look related to bug reported in the ticket